### ISSUE TITLE
Core wallet balance error

### ISSUE THREAD

**Issue by larrysalibra** [blockstack/blockstack-core#619]
After sending all of the regtest bitcoin from the core wallet to another address, querying the core wallet balance gives the following error:

```
[2017-09-26 21:26:51,012] [DEBUG] [registrar:951] (4573.123145401356288) Registrar sleeping for 1
[2017-09-26 21:26:52,034] [DEBUG] [registrar:951] (4573.123145401356288) Registrar sleeping for 1
[2017-09-26 21:26:53,049] [DEBUG] [registrar:951] (4573.123145401356288) Registrar sleeping for 1
[2017-09-26 21:26:54,065] [DEBUG] [registrar:951] (4573.123145401356288) Registrar sleeping for 1
[2017-09-26 21:26:54,481] [DEBUG] [rpc:3889] (4573.123145405562880) 
full path: /v1/wallet/balance/0
method: GET
path: /v1/wallet/balance/0
qs: {}
headers:
origin: http://localhost:3000
accept-language: en-US,en;q=0.8
accept-encoding: gzip, deflate, br
connection: keep-alive
accept: */*
user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.113 Safari/537.36
host: localhost:6270
referer: http://localhost:3000/wallet/send-core
authorization: bearer blockstack_integration_test_api_password

[2017-09-26 21:26:54,482] [DEBUG] [app:114] (4573.123145405562880) Not a valid token
[2017-09-26 21:26:54,482] [DEBUG] [rpc:3933] (4573.123145405562880) Authenticated with password
[2017-09-26 21:26:54,513] [ERROR] [blockchain:211] (4573.123145405562880) -4: The wallet already contains the private key for this address or script
Traceback (most recent call last):
  File "/Users/larry/git/blockstack-testing/develop/blockstack-venv/lib/python2.7/site-packages/blockstack-0.16.0.0-py2.7.egg/blockstack_client/backend/blockchain.py", line 209, in get_utxos
    data = tx_get_unspents( address, utxo_client )
  File "/Users/larry/git/blockstack-testing/develop/blockstack-venv/lib/python2.7/site-packages/blockstack-0.16.0.0-py2.7.egg/blockstack_client/scripts.py", line 381, in tx_get_unspents
    data = get_unspents(address, utxo_client)
  File "/Users/larry/git/blockstack-testing/develop/blockstack-venv/lib/python2.7/site-packages/blockstack-0.16.0.0-py2.7.egg/blockstack_client/utxo.py", line 439, in get_unspents
    return bitcoind_utxo_get_unspents(address, blockchain_client)
  File "/Users/larry/git/blockstack-testing/develop/blockstack-venv/lib/python2.7/site-packages/blockstack-0.16.0.0-py2.7.egg/blockstack_client/backend/utxo/bitcoind_utxo.py", line 100, in get_unspents
    bitcoind.importaddress(str(address))
  File "/Users/larry/git/blockstack-testing/develop/blockstack-venv/lib/python2.7/site-packages/virtualchain/lib/blockchain/bitcoin_blockchain/authproxy.py", line 141, in __call__
    raise JSONRPCException(response['error'])
JSONRPCException: -4: The wallet already contains the private key for this address or script
[2017-09-26 21:26:54,514] [DEBUG] [blockchain:212] (4573.123145405562880) Failed to get UTXOs for mvF2KY1UbdopoomiB371epM99GTnzjSUfj
[2017-09-26 21:26:54,515] [ERROR] [blockchain:310] (4573.123145405562880) Failed to get UTXOs for mvF2KY1UbdopoomiB371epM99GTnzjSUfj: Failed to get UTXOs for mvF2KY1UbdopoomiB371epM99GTnzjSUfj
[2017-09-26 21:26:54,515] [ERROR] [actions:421] (4573.123145405562880) Failed to get balance
[2017-09-26 21:26:54,516] [DEBUG] [rpc:2219] (4573.123145405562880) Failed to query wallet balance: Failed to get balance for mvF2KY1UbdopoomiB371epM99GTnzjSUfj
127.0.0.1 - - [26/Sep/2017 21:26:54] "GET /v1/wallet/balance/0 HTTP/1.1" 503 -
```

**Comment by larrysalibra** [comment]
github doesn't like bzip2 :-( 


[api_endpoint 2.log.zip](https://github.com/blockstack/blockstack-core/files/1333307/api_endpoint.2.log.zip)

**Comment by kantai** [comment]
Tested and fixed in `develop` as of PR #620 (added a test for the bug, and it now passes) 

Thanks for reporting this @larrysalibra
